SharpBucket/SharpBucket.csproj

```diff
@@ -1,171 +1,14 @@
-    <Compile Include="V2\Pocos\Merge.cs" />
-    <Compile Include="V2\Pocos\PullRequest.cs" />
-    <Compile Include="V2\Pocos\PullRequestInfo.cs" />
-    <Compile Include="V2\Pocos\PullRequestsInfo.cs" />
-    <Compile Include="V2\Pocos\Repository.cs" />
-    <Compile Include="V2\Pocos\Owner.cs" />
-    <Compile Include="V2\Pocos\Source.cs" />
-    <Compile Include="V2\Pocos\User.cs" />
-    <Compile Include="V2\EndPoints\RepositoriesEndPoint.cs" />
-    <Compile Include="V2\Pocos\TeamProfile.cs" />
-    <Compile Include="V2\Pocos\Team.cs" />
-    <Compile Include="V2\Pocos\Watcher.cs" />
-    <Compile Include="V2\Pocos\WatcherInfo.cs" />
-    <Compile Include="V2\SharpBucketV2.cs" />
-    <Compile Include="V1\Pocos\IssueFollowers.cs" />
-    <Compile Include="SharpBucket.cs" />
-    <Compile Include="V2\EndPoints\TeamsEndPoint.cs" />
-    <Compile Include="V2\EndPoints\UsersEndpoint.cs" />
-    <Compile Include="V1\EndPoints\IssuesResource.cs" />
-    <Compile Include="V1\EndPoints\RepositoriesEndPoint.cs" />
-    <Compile Include="V1\EndPoints\UserEndPoint.cs" />
-    <Compile Include="V1\EndPoints\UsersEndPoint.cs" />
-    <Compile Include="Authentication\IAuthenticate.cs" />
-    <Compile Include="V1\Pocos\BranchInfo.cs" />
-    <Compile Include="V1\Pocos\Changeset.cs" />
-    <Compile Include="V1\Pocos\ChangesetInfo.cs" />
-    <Compile Include="V1\Pocos\Comment.cs" />
-    <Compile Include="V1\Pocos\Component.cs" />
-    <Compile Include="V1\Pocos\Consumer.cs" />
-    <Compile Include="V1\Pocos\Diffstat.cs" />
-    <Compile Include="V1\Pocos\DiffstatInfo.cs" />
-    <Compile Include="V1\Pocos\EmailInfo.cs" />
-    <Compile Include="V1\Pocos\EventData.cs" />
-    <Compile Include="V1\Pocos\EventInfo.cs" />
-    <Compile Include="V1\Pocos\FileInfo.cs" />
-    <Compile Include="V1\Pocos\Filter.cs" />
-    <Compile Include="V1\Pocos\InvitationsInfo.cs" />
-    <Compile Include="V1\Pocos\Issue.cs" />
-    <Compile Include="V1\Pocos\IssuesInfo.cs" />
-    <Compile Include="V1\Pocos\Metadata.cs" />
-    <Compile Include="V1\Pocos\Milestone.cs" />
-    <Compile Include="V1\Pocos\Privileges.cs" />
-    <Compile Include="V1\Pocos\RepositoriesOverview.cs" />
-    <Compile Include="V1\Pocos\Repository.cs" />
-    <Compile Include="V1\Pocos\RepositoryEvent.cs" />
-    <Compile Include="V1\Pocos\SSH.cs" />
-    <Compile Include="V1\Pocos\SSHDetailed.cs" />
-    <Compile Include="V1\Pocos\Tag.cs" />
-    <Compile Include="V1\Pocos\User.cs" />
-    <Compile Include="V1\Pocos\UserInfo.cs" />
-    <Compile Include="V1\Pocos\Version.cs" />
-    <Compile Include="V1\Pocos\Wiki.cs" />
-    <Compile Include="Properties\AssemblyInfo.cs" />
-    <Compile Include="V1\SharpBucketV1.cs" />
+    <PackageReference Include="Microsoft.CSharp" Version="4.4.1" />
+    <PackageReference Include="Newtonsoft.Json" Version="11.0.2" />
+    <PackageReference Include="RestSharp" Version="106.2.1" />
   </ItemGroup>
   <ItemGroup>
-    <None Include="packages.config" />
```

Comment:
this file should be deleted from the repository (not just not listed in the csproj)